tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -1109,6 +1109,11 @@ def test_chunk_iterator_stride(self):
         self.assertEqual(len(outs), 2)
         self.assertEqual([o["stride"] for o in outs], [(90, 0, 0), (30, 20, 0)])
         self.assertEqual([o["input_values"].shape for o in outs], [(1, 90), (1, 30)])
+        
+        outs = list(chunk_iter(inputs, feature_extractor, 36, 6, 6, ratio))
```

Comment:
Roughly scaled down params from the example in the [issue](https://github.com/huggingface/transformers/issues/21568)